parser/src/sourcemanager.cpp

```diff
@@ -221,6 +234,30 @@ void SourceManager::updateFile(const model::File& file_)
     });
 }
 
+void SourceManager::removeFile(const model::File& file_)
+{
+  bool removeContent = false;
+
+  // Delete File and FileContent (only when no other File references it)
+  _transaction([&, this]() {
+    auto relFiles = _db->query<model::File>(
+        odb::query<model::File>::content == file_.content.object_id());
+    if(std::distance(relFiles.begin(), relFiles.end()) == 1)
```

Comment:
Space missing before paren. Also, [`odb::result<T>`](https://www.codesynthesis.com/products/odb/doc/manual.xhtml#4.4) has a `size()` getter.

Reply:
Fixed in 64cb407581662e78675baf916a95dba2b765b3e4.